### ISSUE TITLE
missing 1 required argument "num_spatial_dims"

### ISSUE THREAD

**Issue by wanghao-cst** [mit-han-lab/litepose#28]
Hi, when I run the first command to train the supermobile model, threre is an error as shown in the pict. Maybe it is caused by the wrong torch version, my version is 1.12, could u ple have a check?
![微信图片_20221103095942](https://user-images.githubusercontent.com/55015183/199634723-2081aad9-7f1c-4dfd-b388-c5cd539414ee.jpg)

**Comment by random9v2** [comment]
Try replacing [this line](https://github.com/mit-han-lab/litepose/blob/main/lib/models/layers/super_layers.py#L40) by:
`output_padding = self._output_padding(x, output_size, self.stride, self.padding, self.kernel_size, num_spatial_dims=2)`

It's due to a PyTorch update, at some point the argument wasn't required and 2D the default as far as I remember.

**Comment by wanghao-cst** [comment]
> 

Thx, that works. may I know your torch version?

**Comment by random9v2** [comment]
I'm using some nightlies, 1.13.x 1.14.x, that's why I had to adjust that, too. I never trained the supernet though, so please apply that with caution. With subnets, all works as expected on my end.

**Comment by suiyingliuxin** [comment]
Why have you never trained a super network? If the network structure is modified, wouldn't it be necessary to start training the super network from scratch?
